### ISSUE TITLE
Accessible chart tab panel

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#961]
Some feedback from DAC:

> I have just asked a screen reader to test the line charts in the chart tab panels (<canvas> with the role of ‘application’). In reading mode JAWS bypasses the application completely and does not read the label unless tab is used to focus the <canvas>, this means that the text alternative is not relayed. Some visually hidden fallback text may help users understand that there is content there in reading mode. I don’t know the exact logistics of this myself, I have just quickly tried to do an internet search and text included in the <canvas> tag appears to screen reader users when I just tested, and pressing enter then brought the application into focus when I tested in Chrome i.e. <canvas>line chart press enter to browse data points with left and right arrow keys</canvas>
> 
> Another thing that came up was that he said ‘use arrow keys’ was not a useful instruction as he automatically uses ‘up and down’ keys, so if ‘left and right’ were specified this would be easier. 

@brockfanning as I've never used Jaws I'm not sure what this means - I think the main thing I can see they're suggesting is to add the text in the canvas tag? Happy to ask for clarification though

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin Just to avoid re-work, it might be helpful to ask exactly what text (words) should go into the canvas tag.

One other request I see here is to specify "left and right arrows" instead of just "arrows", which is an easy change to make.

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin Your PR seems good to me. I don't totally understand why the duplicate fallback text is needed, so my only worry is that some screenreaders will read the text twice. But I've got no problem approving, so will do that now.
